src/features/game/expansion/lib/boosts.ts

```diff
@@ -131,19 +133,39 @@ export const getCookingTime = (
  * @param bumpkin to check for skills
  * @returns boosted food exp
  */
-export const getFoodExpBoost = (foodExp: number, bumpkin: Bumpkin): number => {
-  let boostedExp = new Decimal(foodExp);
+export const getFoodExpBoost = (food: Consumable, bumpkin: Bumpkin): number => {
+  let boostedExp = new Decimal(food.experience);
   const { skills, equipped } = bumpkin;
   const { tool } = equipped;
 
   //Bumpkin Skill Boost Kitchen Hand
   if (skills["Kitchen Hand"]) {
     boostedExp = boostedExp.mul(1.05);
   }
+  //Bumpkin Skill Boost Curer
+  if (food.building === "Deli" && skills["Curer"]) {
+    boostedExp = boostedExp.mul(1.15);
+  }
+
   //Bumpkin Wearable Boost Golden Spatula
   if (tool === "Golden Spatula") {
     boostedExp = boostedExp.mul(1.1);
   }
 
   return boostedExp.toNumber();
 };
+
+export const getOrderSellPrice = (bumpkin: Bumpkin, order: GrubShopOrder) => {
+  const { skills } = bumpkin;
+  let mul = 1;
+
+  if (skills["Michelin Stars"]) {
+    mul += 0.05;
+  }
+
+  if (order.name in CAKES() && bumpkin.equipped.coat == "Chef Apron") {
```

Comment:
Can we please use `===` here. Because Javascript equality is so random we should always use strict equality.

Reply:
> Can we please use `===` here. Because Javascript equality is so random we should always use strict equality.


True, how did i miss that 😭